docs/Technical-Documentation/Architecture-Decision-Record/008-deployment-flow.md

```diff
@@ -46,4 +49,4 @@ Within the dev space:
 
 ## Notes
 
-- As of December 2021, merges into `HHS/TANF-app:main` do not deploy to any environment, since we are pending activation of the production space (#721). 
\ No newline at end of file
+- As of December 2021, merges into `HHS/TANF-app:main` do not deploy to any environment, since we are pending activation of the production space (#721). 
```

Comment:
```suggestion
- As of April 2022, merges into `HHS/TANF-app:master` do not deploy to any environment, since we are pending activation of the production space (#895). 
```